homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -65,99 +73,99 @@ def discover_devices(device_id: int) -> List[Tuple[str, str]]:
     return result
 
 
-async def see_device(
-    hass: HomeAssistantType, async_see, mac: str, device_name: str, rssi=None
-) -> None:
-    """Mark a device as seen."""
-    attributes = {}
-    if rssi is not None:
-        attributes["rssi"] = rssi
+def lookup_name(mac: str) -> Optional[str]:
+    """Lookup a Bluetooth device name."""
+    _LOGGER.debug("Scanning %s", mac)
+    return bluetooth.lookup_name(mac, timeout=5)
 
-    await async_see(
-        mac=f"{BT_PREFIX}{mac}",
-        host_name=device_name,
-        attributes=attributes,
-        source_type=SOURCE_TYPE_BLUETOOTH,
-    )
 
+async def load_initial_entities(
+    hass: HomeAssistantType,
+    config_entry: ConfigEntry,
+    async_add_entities: Callable[List[Any]],
+) -> Set[str]:
+    """Load tracked entities from device registry."""
+    device_registry_instance = await device_registry.async_get_registry(hass)
+    macs: Set[str] = {
+        mac
+        for device in device_registry_instance.devices.values()
+        for domain, mac in device.identifiers
+        if domain == DOMAIN
+    }
 
-async def get_tracking_devices(hass: HomeAssistantType) -> Tuple[Set[str], Set[str]]:
-    """
-    Load all known devices.
+    entities = []
+    for mac in macs:
+        hass.data[DOMAIN]["devices"].add(mac)
```

Comment:
Do we use this stored set anywhere else? We might need it later when we add the component module.

Reply:
I don't think so